src/backend/vulkan/src/native.rs

```diff
@@ -64,13 +63,19 @@ pub struct Sampler(pub vk::Sampler);
 #[derive(Debug, Hash)]
 pub struct RenderPass {
     pub raw: vk::RenderPass,
-    pub clear_attachments_mask: u64,
+    pub attachment_count: usize,
 }
 
-#[derive(Debug, Hash)]
-pub struct Framebuffer {
-    pub(crate) raw: vk::Framebuffer,
-    pub(crate) owned: bool,
+pub type FramebufferKey = SmallVec<[vk::ImageView; 5]>;
```

Comment:
There might be some value in the future of not hardcoding the `5`, maybe by getting `surface_capabilities.max_image_count/min_image_count`.
Hardcoded also in command.rs -> line: 252:
 `let mut raw_clear_values = SmallVec::<[vk::ClearValue; 5]>::new();`

Reply:
Added a constant for this. We may come back to it and set up a real maximum bound, so that we can use `ArrayVec` instead of `SmallVec`.